### PR TITLE
Cleanup NFT template

### PR DESCRIPTION
**Pull request by 0xaptosj**
Copy the contract code to newly setup digital asset template directory.

### CODE REVIEW

File: templates/digital-asset-template/move/Move.toml [hunk 1]
```diff
@@ -1,9 +1,19 @@
...
+version = "1.0.0"
+authors = []
+
 [addresses]
-module_addr='_'
\ No newline at end of file
+launchpad_addr = "_"
+minter = "_"
+
+[dev-addresses]
```
Comment (0xmaayan) [inline templates/digital-asset-template/move/Move.toml:10]: I found this annoying when tryin to run different move commnads (npm run move:compile/publish) - was trying to fix it on the sdk side to accept optional params like `--dev` but then it introduces some other errors because of this field - thoughts on just removing it for now?

File: templates/digital-asset-template/move/Move.toml [hunk 2]
```diff
@@ -1,9 +1,19 @@
...
-subdir = 'aptos-move/framework/aptos-framework'
+name = "digital_asset_launchpad"
+version = "1.0.0"
+authors = []
+
 [addresses]
-module_addr='_'
\ No newline at end of file
+launchpad_addr = "_"
+minter = "_"
```
Comment (0xmaayan) [inline templates/digital-asset-template/move/Move.toml:8]: what is it for? if required, we need to modify the move scripts probably

File: templates/digital-asset-template/move/sources/launchpad.move [hunk 3]
```diff
@@ -1,4 +1,4 @@
-module launchpad_addr::nft_launchpad {
+module launchpad_addr::launchpad {
```
Comment (0xmaayan) [inline templates/digital-asset-template/move/sources/launchpad.move:1]: why this change?

File: templates/digital-asset-template/sh_scripts/upgrade.sh
Comment (0xmaayan) [inline templates/digital-asset-template/sh_scripts/upgrade.sh:1]: can we get rid of those sh scripts and use the existing ts based scripts?

File: templates/digital-asset-template/contract_address.txt
Comment (0xmaayan) [inline templates/digital-asset-template/contract_address.txt:1]: we dont need it with the tool setup

**Review by 0xmaayan — COMMENTED** [review]
some things are not needed - have you ran the wizard with this template locally?

File: templates/digital-asset-template/move/Move.toml (hunk 2 above)
Comment (0xaptosj) [reply templates/digital-asset-template/move/Move.toml:8]: this is required, it's like declaring all the addresses needed for the contract, `launchpad_addr` is address of contract itself, `minter` is address of dependency contract.

File: templates/digital-asset-template/move/Move.toml (hunk 1 above)
Comment (0xaptosj) [reply templates/digital-asset-template/move/Move.toml]: removed

File: templates/digital-asset-template/move/sources/launchpad.move (hunk 3 above)
Comment (0xaptosj) [reply templates/digital-asset-template/move/sources/launchpad.move:1]: the package name is digital_asset_launchpad, want to keep it consistent, though we still call it nft inside the contract.. but i tried to not call it nft in module name.

File: templates/digital-asset-template/move/Move.toml (hunk 2 above)
Comment (0xmaayan) [reply templates/digital-asset-template/move/Move.toml:8]: > minter is address of dependency contract

then should it be a default address? or is it network based?

Comment (0xaptosj) [reply templates/digital-asset-template/move/Move.toml:8]: it's network based

**Comment by 0xaptosj** [comment]
shell script removed
